src/filesystem.rs

```diff
@@ -34,31 +34,41 @@ fn follow_symlink(pathbuf: PathBuf) -> PathBuf {
     }
 }
 
-fn exe_pathbuf() -> PathBuf {
-    let pathbuf = std::env::current_exe().unwrap();
-    follow_symlink(pathbuf)
-}
-
 pub fn cheat_pathbuf() -> Option<PathBuf> {
-    let exe_parent_str = exe_parent_string();
+    match dirs::config_dir() {
+        Some(mut d) => {
+            d.push("navi");
+            Some(d)
+        }
+        None => None,
+    }
+}
 
-    let array = ["cheats", "../libexec/cheats", "../cheats", "../../cheats"];
-    for elem in &array {
-        let p = format!("{}/{}", exe_parent_str, elem);
-        let meta = metadata(&p);
-        if let Ok(m) = meta {
-            if m.is_dir() {
-                return Some(PathBuf::from(p));
-            }
+pub fn bundled_cheat_pathbuf() -> Option<PathBuf> {
+    match dirs::data_dir() {
```

Comment:
I'm afraid there's no way for the brew formula to save files to `dirs::data_dir()` 😞

Reply:
Ah that's a shame!

I'm not sure if it's considered good practice or not but we could have 2 versions of that function and only compile one for macos and one for not macos using the cfg attributes?

Something like 

```
#[cfg(target_os = "macos")]
pub fn bundled_cheat_pathbuf() -> Option<PathBuf> {
...
}

#[cfg(not(target_os = "macos"))]
pub fn bundled_cheat_pathbuf() -> Option<PathBuf> {
...
}
```